sched/environ/environ.h

```diff
@@ -33,8 +33,10 @@
  ****************************************************************************/
 
 #ifdef CONFIG_DISABLE_ENVIRON
-#  define env_dup(group)     (0)
-#  define env_release(group) (0)
+#  define env_dup(group)       (0)
+#  define env_set(group, envp) (0)
+#  define env_release(group)   (0)
+#  define env_get(group)       (NULL)
```

Comment:
why not use environ or get_environ_ptr directly?

Reply:
I did not know it exists, I will change

Reply:
Done